Makefile

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+MK_DIR :=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+DISTRO ?= clearlinux
```

Comment:
I am adding two linux Distributions to show that the Guest OS can be cased in any system as far as it complies with our system requirements.

I added two rpm based Linux distributions because I know how to configure dnf.
I can add a deb based linux distribution as well.
@bergwolf I took a look at image script from hyperstart it is based in busybox, we can either or built it use a busybox/alpine docker image. Then install the agent on top of it as init process
It make sense to support a LTS distro: The need updates would be less frequents. I can add something based in a LTS image.

Reply:
There is no systemd in busybox/alpine, so we'd better use a LTS distro. It is good to see we can support multiple distros as rootfs base, but we should use a LTS distro as the default IMHO.

Reply:
@bergwolf, Hi, I added centOS as LTS distro, I am looking how to add a deb based distro for a next PR.